src/backend/main.ts

```diff
@@ -700,31 +700,19 @@ ipcMain.handle('getLatestReleases', async () => {
   }
 })
 
-ipcMain.on('clearCache', () => {
+ipcMain.on('clearCache', (event) => {
   clearCache()
-  dialog.showMessageBox(mainWindow, {
-    title: i18next.t('box.cache-cleared.title', 'Cache Cleared'),
-    message: i18next.t(
-      'box.cache-cleared.message',
-      'Heroic Cache Was Cleared!'
-    ),
-    buttons: [i18next.t('box.ok', 'Ok')]
-  })
+  event.sender.send(
+    'showDialog',
+    i18next.t('box.cache-cleared.title', 'Cache Cleared'),
+    i18next.t('box.cache-cleared.message', 'Heroic Cache Was Cleared!'),
+    false,
+    [i18next.t('box.ok', 'Ok')]
+  )
```

Comment:
I would love to have a generic function, else we need to adapt code later if we wanna change stuff.
So we need a `showDialogModalAuto` function, which replaces the `showErrorBoxModalAuto`

Reply:
sure thing. will change